test/integration/local/test_dynamic_endpoint.py

```diff
@@ -77,6 +77,11 @@ def make_invocation_request(data, model_name, content_type='application/json'):
     return json.loads(response.content.decode('utf-8'))
 
 
+def make_list_model_request():
+    response = requests.get(MODELS_URL)
+    return json.loads(response.content.decode('utf-8'))
```

Comment:
[Minor] Is there a constant in Python that matches "utf-8"

Reply:
I replaced it with `encodings.utf_8.getregentry().name`.